### PR TITLE
feat:Add switch/case blocks generation

### LINKED ISSUE #55: Extend code generation
Add next constructions:
- function call
- "dynamic" function call
- `switch-case` (optional)

### PR DESCRIPTION
**Pull request by Ivan-Pokhabov**
Resolve #55 
Add switch-case construction

### CODE REVIEW

File: src/generators/code_gen.py [hunk 1]
```diff
@@ -177,6 +177,53 @@ def render(self, visitor: Visitor) -> None:
...
+    """
+    Class of representation switch-case construction block.
+
+    Attributes
+    --------
+    expression : ApplyBinOperator
+        expression in switch
+    case_blocks : List[List[Block]]
+        list of blocks in each case
+    cases : List[int]
```
Comment (vkutuev) [inline src/generators/code_gen.py:190]: Please read [Sphinx docstring style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html). We can replace Sphinx style with Google style, but it is bad to mix them.

Comment (vkutuev) [reply src/generators/code_gen.py:190]: Also, types in docstring are redundant and require maintenance

**Review by vkutuev — CHANGES_REQUESTED** [review]

File: src/generators/code_gen.py (hunk 1 above)
Comment (Ivan-Pokhabov) [reply src/generators/code_gen.py]: Ok, I rework it to Sphinx style

File: src/generators/code_gen.py [hunk 2]
```diff
@@ -177,6 +177,48 @@ def render(self, visitor: Visitor) -> None:
         visitor.send("}")
 
 
+class SwitchCaseBlock(Block):
+    """
+    Class of representation switch-case construction block
+
+    :param expr: expression in switch
+    :type expr: ApplyBinOperator
```
Comment (vkutuev) [inline src/generators/code_gen.py:185]: Please remove types. We use type annotations

File: src/generators/code_gen.py [hunk 3]
```diff
@@ -177,6 +177,48 @@ def render(self, visitor: Visitor) -> None:
         visitor.send("}")
 
 
+class SwitchCaseBlock(Block):
+    """
```
Comment (vkutuev) [inline src/generators/code_gen.py:181]: Summary block should start directly after `"""`.

**Review by vkutuev — APPROVED** [review]

**Pull request merged**